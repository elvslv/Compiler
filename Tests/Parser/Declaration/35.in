type
  TArray = array[1..10] of integer;

var a: TArray;

type
  TRecordArray = record mass: TArray; a, b, c: integer; end;

var b, c: TRecordArray;

const MaxN = 1000;