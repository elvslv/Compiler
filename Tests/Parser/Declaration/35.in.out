type TARRAY = array [const 1..const 10] of INTEGER;
var A : TARRAY;
type TRECORDARRAY = record 
    var A : INTEGER;
    var B : INTEGER;
    var C : INTEGER;
    var MASS : TARRAY;
end;
var B : TRECORDARRAY;
var C : TRECORDARRAY;
const MAXN = 1000; 
